docs/user-guide/usage/cli.md

```diff
@@ -1,256 +1,103 @@
-
-### `--disable-default-ignores, --di`
-
-Allow linting of `node_modules`.
-
-### `--cache`
-
-Store the info about processed files in order to only operate on the changed ones the next time you run stylelint. By default, the cache is stored in `./.stylelintcache`. To adjust this, use `--cache-location`.
-
-Default: `false`.
-
-### `--cache-location`
-
-Path to a file or directory to be used for the cache location.
-
-Default is `"./.stylelintcache"`. If a directory is specified, a cache file will be created inside the specified folder, with a name derived from a hash of the current working directory.
-
-If the directory for the cache does not exist, make sure you add a trailing `/` on *nix systems or `\\` on Windows. Otherwise the path will be assumed to be a file.
-
-### `--formatter, -f`
-
-The output formatter. Options are:
-
--   `compact`
--   `json`
--   `string` (default)
--   `unix`
--   `verbose`
-
-### `--custom-formatter`
-
-Path to a JS file exporting a custom formatting function.
-
-### `--quiet, -q`
-
-Only register violations for rules with an "error"-level severity (ignore "warning"-level).
-
-### `--color, --no-color`
-
-Force enabling/disabling of color.
-
-### `--report-needless-disables, --rd`
-
-Also report errors for stylelint-disable comments that are not blocking a lint warning.
-
-The process will exit with code `2` if needless disables are found.
-
-### `--report-invalid-scope-disables, --risd`
-
-Report stylelint-disable comments that used for rules that don't exist within the configuration object.
-
-The process will exit with code `2` if invalid scope disables are found.
-
-### `--max-warnings, --mw`
-
-Number of warnings above which the process will exit with code `2`.
-
-Useful when setting `defaultSeverity` to `"warning"` and expecting the process to fail on warnings (e.g. CI build).
-
-### `--output-file, -o`
-
-Path of file to write report.
-
-### `--version, -v`
-
-Show the currently installed version of stylelint.
-
-### `--allow-empty-input, --aei`
+```shell
+stylelint "foo/*.css" --config bar/mySpecialConfig.json --output-file myTestReport.txt
+```
 
-When glob pattern matches no files, the process will exit without throwing an error.
+### Example F - multiple globs and specifying a config
 
-## Syntax errors
+Using `bar/mySpecialConfig.json` as config to lint all `.css` files in the `foo` directory and any of its subdirectories and also all `.css` files in the `bar directory`:
 
-The CLI informs you about syntax errors in your CSS.
-It uses the same format as it uses for linting violations.
-The error name is `CssSyntaxError`.
+```shell
+stylelint "foo/**/*.css" "bar/*.css" --config bar/mySpecialConfig.json
+```
 
 ## Exit codes
 
 The CLI can exit the process with the following exit codes:
 
 -   1: Something unknown went wrong.
--   2: At least one rule with an "error"-level severity triggered at least one violations.
+-   2: There was at least one rule or CLI flag error
```

Comment:
[IMO] I think it natural to handle "CLI flag error" with exit code `1`, not `2`. But this description fits the current stylelint behavior... So, this may be another issue. 🤔 

Also, I feel that the removed word "violation" is necessary for clarity.

```diff
-There was at least one rule.
+There was at least one rule violation.
```